server/src/main/java/org/elasticsearch/health/metadata/HealthMetadata.java

```diff
@@ -142,6 +174,8 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             builder.field(FLOOD_STAGE_WATERMARK_FIELD.getPreferredName(), describeFloodStageWatermark());
             builder.field(FROZEN_FLOOD_STAGE_WATERMARK_FIELD.getPreferredName(), describeFrozenFloodStageWatermark());
             builder.field(FROZEN_FLOOD_STAGE_MAX_HEADROOM_FIELD.getPreferredName(), frozenFloodStageMaxHeadroom);
+            builder.field(HIGH_MAX_HEADROOM_FIELD.getPreferredName(), highMaxHeadroom);
+            builder.field(FLOOD_STAGE_MAX_HEADROOM_FIELD.getPreferredName(), floodStageMaxHeadroom);
```

Comment:
Nit: In this case you could change the order of the fields to follow the intuitive one. I think that's the case because:
- There is no parser that is expecting a specific order within elasticsearch, and
- For the `_cluster/state` API we do not give any guarantees to users (if I am not mistaken).

Reply:
Will be done in a next commit. Thanks!